src/Components/Search/SearchBar.tsx

```diff
@@ -1,104 +1,271 @@
-          {({ relayEnvironment }) => {
-            return (
-              <QueryRenderer<SearchBarQuery>
-                environment={relayEnvironment}
-                query={graphql`
-                  query SearchBarQuery($term: String!) {
-                    viewer {
-                      ...SuggestionsSearch_viewer @arguments(term: $term)
-                    }
-                  }
-                `}
-                variables={{ term }}
-                render={({ props }) => {
-                  if (props) {
-                    return (
-                      <SearchSuggestions term={term} viewer={props.viewer} />
-                    )
-                  } else {
-                    return (
-                      <SpinnerContainer>
-                        <Spinner />
-                      </SpinnerContainer>
-                    )
-                  }
-                }}
-              />
-            )
-          }}
-        </ContextConsumer>
-      )
+  onFocus = () => {
+    this.setState({ focused: true })
+  }
+
+  onBlur = () => {
+    this.setState({ focused: false })
+  }
+
+  onSuggestionsClearRequested = () => {
+    this.setState({ input: "", entityID: null, entityType: null })
+  }
+
+  // Navigate to selected search item.
+  onSuggestionSelected = (
+    _e,
+    {
+      suggestion: {
+        node: { href },
+      },
     }
+  ) => {
+    window.location.href = href
   }
 
-  render() {
-    const { term } = this.state
+  renderPreview() {
+    const { entityID, entityType } = this.state
+    if (entityID && entityType) {
+      return <SearchPreview entityID={entityID} entityType={entityType} />
+    }
+  }
+
+  renderSuggestionsContainer = ({ containerProps, children, query }) => {
+    const { focused } = this.state
+
+    let firstItem = null
+    if (query) {
+      firstItem = <Box>Search "{query}"</Box>
+    } else if (focused) {
+      firstItem = <Box>{PLACEHOLDER}</Box>
+    }
 
     return (
-      <>
-        <Input
-          block
-          onInput={this.searchTextChanged.bind(this)}
-          onPaste={this.searchTextChanged.bind(this)}
-          onCut={this.searchTextChanged.bind(this)}
-          autoFocus
+      <Box {...containerProps}>
+        <Flex flexDirection={["column", "row"]}>
+          <Box width={["100%", "50%"]}>
+            <Flex flexDirection="column">
+              {firstItem}
+              {children}
+            </Flex>
+          </Box>
+          <Box width={["100%", "50%"]}>{this.renderPreview()}</Box>
+        </Flex>
+      </Box>
+    )
+  }
+
+  renderSuggestion = (
+    { node: { displayLabel, searchableType } },
+    { isHighlighted }
+  ) => {
+    return (
+      <Box style={{ backgroundColor: isHighlighted ? "#ddd" : "#fff" }}>
+        <SuggestionItem display={displayLabel} label={searchableType} />
+      </Box>
+    )
+  }
+
+  renderInputComponent = inputProps => (
+    <Box>
+      <Input style={{ width: "100%" }} {...inputProps} />
```

Comment:
Can we use the new Input that @pepopowitz created?

Reply:
Sure! This (and a couple previous PR's) were sort of spike-y in that they were more proof-of-concept of using the autosuggest lib, and wiring up functionality, and _then_ we're moving to more UI passes.

This is actually the reverse order we did things for the artist/artwork pages (where we _started_ with UI and then added functionality), but we actually discussed that as a team this go around and decided for this project to rough in all the pieces and functionality and then improve the UI and layout. Apologies for the non design system-y spike-y stuff! That should [hopefully] all be iterated on now.

Reply:
The new Input hasn't been moved to palette yet, so it's not yet design-system-friendly. We [got specs on it today](https://github.com/artsy/palette/issues/253), though! We'll make sure to address this when we get there, if it hasn't been already.